octopoes/octopoes/models/ooi/config.py

```diff
@@ -0,0 +1,21 @@
+from typing import Literal
+
+from octopoes.models import OOI, Reference
+from octopoes.models.persistence import ReferenceField
+
+
+class Config(OOI):
+    object_type: Literal["Config"] = "Config"
+
+    ooi: Reference = ReferenceField(OOI)
```

Comment:
Optionally we might want to include a reference to the originating source for this config, eg, the questionOOI that @Donnype is creating.

Reply:
Yes we could! My thought process was that that OOI would be visible in the "observed by" table

Reply:
Hm, but this configOOI can be the result of a normaliser reading/parsing the data inserted by a user. The observed by table is meant to display the origin OOI's when this OOI was created from a Bit right?

Reply:
Obeserved by - when the ooi is created from a normlizer (other then manual ooi normalizer)
Declared by - when the ooi is created from the manual ooi normalizer
Infered by - when the ooi is created from a bit

Reply:
So when the OOI is created from the question ooi, it will show in the observed by table